### DISCUSSION TITLE
Worked this morning, now the scraper no longer presents list of subscriptions...

### DISCUSSION THREAD

**Discussion by paulescobar** [UltimaHoarder/UltimaScraper#637] [Q&A] [answered]
I'm not sure if this is a bug, or I'm doing something wrong. This morning I setup the scraper and used it on individual OF subs. It worked perfectly. Now all of a sudden, it won't load the list of subs.

So when you start the app, it says
`Site: 0 = OnlyFans | x = Exit`

Before, when I pressed "0" it would have the options that let me advance and choose the OF profile for scraping:
```
Choose Profile
Names: Username = username | 0 = all | 1 = Default
```

But now it simply shows scraping status messages, then ends with "There's nothing to scrape":
```
Choose Profile
Scraping Paid Content
Scraping Subscriptions
There's nothing to scrape.
Archive Completed in 0.0 minutes
```

I thought maybe my cookie information needed to be updated, but that didn't solve the problem. What could be happening here?

**Comment by paulescobar** [comment, marked as answer]
Reinstalled it, now it works again. Very strange. Not sure if I screwed up something in the config file earlier or what.

**Comment by shixel** [comment]
Getting this same error. I was scraping just fine until the website forced me to log back in.
After updating the config data it says there's nothing to scrape.

**Reply by shixel** [reply]
Figured it out. I had set the "auto_profile_choice" to "a" instead of "default" in the config.

**Comment by drickerby** [comment]
Getting the same thing as OP. Cleared pip cache, ensured all my cookie settings are correct, updated the requirements and have the latest version of the script (6.2.3) but just get:

`Site: 0 = onlyfans | x = Exit
0
Session IP: 89.238.176.214

Choose Profile
Scraping Paid Content
Scraping Subscriptions
There's nothing to scrape.
Archive Completed in 0.0 Minutes
Pausing scraper for 0 seconds.
Site: 0 = onlyfans | x = Exit`

**Reply by drickerby** [reply]
Never mind. For some reason "active": false was at the bottom of my default profile's auth.json. No idea why, I didn't set that. Those config files should be read-only if they're user edited.

**Reply by jp1151419** [reply]
Yeah them being switched to false when something goes wrong is tricky cause you have to go back and make sure they're fine after each attempt to fix the issue.
